tests/unit/source_code/test_functional.py

```diff
@@ -62,6 +62,13 @@ class Functional:
     )
     _location = Path(__file__).parent / 'samples/functional'
 
+    TEST_DBR_VERSION = {
+        'python-udfs_13_3.py': (13, 3),
```

Comment:
don't you want to introduce `# ucx session state: {"dbr_version": [13, 3]}` / `# ucx session state: {"is_serverless": true}` meta-comments, where you parse JSON and set override the current session state?.. make sure to validate this instruction appears only once per file, because of the way we initialise linters. 

technically, we can make it into processing instrustion and be in the parser flow, but it's more complicated.